src/components/ContentUploader/ContentUploader.js

```diff
@@ -67,6 +67,7 @@ type State = {
 
 const CHUNKED_UPLOAD_MIN_SIZE_BYTES = 52428800; // 50MB
 const FILE_LIMIT_DEFAULT = 100; // Upload at most 100 files at once by default
+const UPLOAD_CONCURRENCY = 6;
 
 class ContentUploader extends Component<Props, State> {
     id: string;
```

Comment:
I think we're documenting all class properties here so `numItemsUploading:number = 0;` probably belongs here

Reply:
done